app/views/ReportView/components/TherapeuticTargets/components/AutocompleteHandler/index.tsx

```diff
@@ -96,20 +92,8 @@ const AutocompleteHandler = (props) => {
   );
 };
 
-AutocompleteHandler.propTypes = {
-  defaultValue: PropTypes.string,
-  type: PropTypes.string.isRequired,
-  label: PropTypes.string.isRequired,
-  required: PropTypes.bool,
-  onChange: PropTypes.func,
-  minCharacters: PropTypes.number,
-};
-
-AutocompleteHandler.defaultProps = {
-  defaultValue: '',
-  required: false,
-  onChange: () => { },
-  minCharacters: 3,
+export {
+  AutocompleteHandler,
 };
 
 export default AutocompleteHandler;
```

Comment:
are both exports needed?

Reply:
not required no, but it is more of a convention thing, just that if we do decide to export more things from this module, we can be more modular about it